api/posters/statistics/helpers.ts

```diff
@@ -10,12 +9,8 @@ export const getCreatedPostersBetween = async (
   lowerBound: Date,
   upperBound: Date,
 ) =>
-  PosterModel.scan({
-    updatedAt: {
-      ge: dateToUnixTimeStamp(lowerBound),
-      le: dateToUnixTimeStamp(upperBound),
-    },
-  })
+  PosterModel.scan('updatedAt')
+    .between(lowerBound.valueOf(), upperBound.valueOf())
```

Comment:
Why don't we use `getTime()` instead? It is [equivalent to valueOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/valueOf#description) and is more readable. What do you think?

Reply:
Sure, makes sense to me :+1: